rivalcfg/helpers.py

```diff
@@ -134,3 +134,17 @@ def uint_to_little_endian_bytearray(number, size):
     for i in range(size):
         nle[i] = number >> i*8 & 0xFF
     return nle
+
+
+def bytes_to_high_low_nibbles(byte):
+    """Splits bytes into high and low nibbles.
+    """
+    high = byte >> 4
+    low = byte & 0x0F
+    return high, low
+
+
+def nibbles_to_byte(nibble1, nibble2):
+    """Converts two nibbles into a single byte.
+    """
+    return (nibble1 << 4) | (nibble2 >> 4)
```

Comment:
I am not sure here.

if we have `0x0f >= nibble2 >= 0x00`, it will always be equal to 0 after the shift operation:

```
>>> 0x0f >> 4                                                                                                                                                                                                
0
```

maybe the operation should be `return (nibble1 << 4) | nibble2`?

(tell me if I made a mistake :) )

Reply:
No its my mistake :) ty